src/vinyldns/client.py

```diff
@@ -622,12 +631,39 @@ def list_batch_change_summaries(self, start_from=None, max_items=None, **kwargs)
             args.append(u'startFrom={0}'.format(start_from))
         if max_items is not None:
             args.append(u'maxItems={0}'.format(max_items))
+        if ignore_access:
+            args.append(u'ignoreAccess={0}'.format(ignore_access))
+        if approval_status:
+            args.append(u'approvalStatus={0}'.format(approval_status))
 
         url = urljoin(self.index_url, u'/zones/batchrecordchanges') + u'?' + u'&'.join(args)
 
         response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
         return ListBatchChangeSummaries.from_dict(data)
 
+    def approve_batch_change(self, batch_change_id, approval=None, **kwargs):
```

Comment:
same comment as reject. i think the username field may just be misnamed (in a GET outside of python, i do see that one, just not in the python client):

```
>>> client.approve_batch_change("6a8177e0-56f4-4777-a855-309f59797640", "fsdf")
<vinyldns.batch_change.BatchChange object at 0x104356cc0>
>>> test = client.get_batch_change("6a8177e0-56f4-4777-a855-309f59797640")
>>> test.status
'Complete'
>>> test.review_comment
>>> test.reviewer_id
'support-user-id'
>>> test.reviewer_username
>>> test.review_timestamp
datetime.datetime(2019, 8, 2, 18, 13, 55, tzinfo=tzutc())
>>> test.approval_status
'ManuallyApproved'
```

Reply:
it's not a string it's a `AcceptBatchChangeRequest` object that takes a string

Reply:
what about reviewer_username?

Reply:
@pauljamescleary can you comment on this? Needing to create an AcceptBatchChangeRequest in order to call approve_batch_change with a comment feels confusing to me. Is there a strong argument to not just accept a string?

Reply:
@britneywright if there are only a few parameters (in this case one), we can just add "comments" to the python method and create the "AcceptBatchChangeRequest" ourselves.

Reply:
updated this to take a string instead of an object. I opted to remove the objects entirely since we don't return them, we return the batch change itself.